AnkiDroid/src/main/java/com/ichi2/async/Connection.java

```diff
@@ -507,7 +507,9 @@ private Payload doInBackgroundSync(Payload data) {
                         data.resultType = USER_ABORTED_SYNC;
                         data.result = new Object[]{e};
                     }
-                    mediaError = AnkiDroidApp.getAppResources().getString(R.string.sync_media_error) + "\n\n" + e.getLocalizedMessage();
+                    mediaError = AnkiDroidApp.getAppResources().getString(R.string.sync_media_error) + "\n\n" + e.getLocalizedMessage()
+                            + "\n\n" + AnkiDroidApp.getAppResources().getString(R.string.sync_media_downloaded_count, mediaClient.getDownloadCount())
+                            + "\n\n" + AnkiDroidApp.getAppResources().getString(R.string.sync_media_uploaded_count, mediaClient.getUploadCount());
```

Comment:
I think one `\n` would be better here

```suggestion
                            + "\n" + AnkiDroidApp.getAppResources().getString(R.string.sync_media_uploaded_count, mediaClient.getUploadCount());
```

Reply:
@david-allison-1 Can you give me a feedback on this

Reply:
Sounds good to me